app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
```

Comment:
Should this be descoped and live in `components/`?

Reply:
I'm trying to keep it conservative for now. Let's descope once we have another component that needs this.

Reply:
OK, cool.